contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -1837,6 +1853,12 @@ pltsql_sequence_datatype_map(ParseState *pstate,
 		ereport(WARNING,
 				(errmsg("NUMERIC or DECIMAL type is cast to BIGINT")));
 	}
+
+	if (aclresult != ACLCHECK_OK)
+    	aclcheck_error_type(aclresult, *newtypid);
+
+    if(getBaseType(*newtypid)!=INT2OID || getBaseType(*newtypid)!=INT4OID || getBaseType(*newtypid)!=INT8OID) *newtypid = getBaseType(*newtypid);
```

Comment:
Fix indentation. It is good coding practice to maintain 80 char limit in one line.